src/modules/InsightsLibrary/components/ViewEdit/ModalSettings/container.js

```diff
@@ -42,6 +43,7 @@ export default class ModalSettingsBuilder {
     return {
       publishState: get(state, 'insightsLibrary.insights.data.publishState', 'DRAFT', 'String'),
       insightId: get(state, 'insightsLibrary.insights.data.id'),
+      isPublishAvl: get(state, 'insightsLibrary.insights.data.study.status.availableActions', []).includes(studyActions.publishPaper),
```

Comment:
Let's use same names for same things. So call it "canPublishPaper" is it is done below